example/disasm/full.py

```diff
@@ -142,15 +142,13 @@
 
 
 # Generate dotty graph
-all_blocs = []
+all_blocs = BasicBlocks()
```

Comment:
After a quick review, there are too many lines to change regarding this correction, lines which are not correlated with the logic of the concerned commit.
Then, I prefer to make a separate commit for all of them, and, a separate PR for the same reasons.

Thanks for your review!